products/google-nexus.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Google Nexus
+category: device
+tags: google mobile-phone tablet
+iconSlug: google
+permalink: /google-nexus
+versionCommand: "Settings -> About Phone -> Regulatory labels"
+releasePolicyLink: https://support.google.com/nexus/answer/11227897
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: true
+discontinuedColumn: true
+eolColumn: Guaranteed Software Updates
+customColumns:
+-   property: supportedAndroidVersions
+    position: after-release-column
+    label: Supported Android # https://en.wikipedia.org/wiki/Google_Nexus
+    description: Supported Android versions range
+    link: https://endoflife.date/android
+
+# Supported Android versions range is based on https://www.gsmarena.com/.
+releases:
+-   releaseCycle: "6p"
+    releaseLabel: "Nexus 6P"
+    releaseDate: 2015-09-29
+    support: 2018-09-28
+    eol: 2018-09-28
```

Comment:
```suggestion
    support: 2017-09-01
    eol: 2018-11-01
```

According to https://support.google.com/nexus/answer/11227897.

- support = No guaranteed Android version updates after
- EOL = No guaranteed security updates after

Used start of the month for both.

Other models must also be updated accordingly.